controller/controller_manager.go

```diff
@@ -106,8 +107,12 @@ func StartControllers(logger logrus.FieldLogger, stopCh chan struct{}, controlle
 		engineImageInformer, nodeInformer, settingInformer,
 		imInformer, shareManagerInformer,
 		backingImageInformer, backingImageManagerInformer, backingImageDataSourceInformer,
+<<<<<<< HEAD
 		backupTargetInformer, backupVolumeInformer, backupInformer,
 		recurringJobInformer,
+=======
+		backupTargetInformer, backupVolumeInformer, backupInformer, supportBundleInformer,
+>>>>>>> update informer
```

Comment:
??? How does this passed CI?

Reply:
OK saw the later commit removes it. Need to merge them and put the code in the corresponding commit.